src/SourceGenerators/Uno.UI.InternalSourceGenerators/Uno.UI.InternalSourceGenerators.csproj

```diff
@@ -0,0 +1,104 @@
+﻿<Project Sdk="MSBuild.Sdk.Extras">
```

Comment:
This file is almost a copy from `Uno.UI.SourceGenerators.csproj`.

I'm not sure if anything here is redundant in the internal generators. (e.g, is `_UnoToolkitOverrideNuget` target needed? etc..

Reply:
> This file is almost a copy from Uno.UI.SourceGenerators.csproj.

No longer true. I cleaned up the csproj for internal generators, making it .NET Standard 2.0 only, and removing unnecessary things.